packages/salesforcedx-apex-debugger/src/core/streamingService.ts

```diff
@@ -100,7 +100,7 @@ export class StreamingService {
     return Promise.resolve(this.isReady());
   }
 
-  private removeTrailingSlashURL(instanceUrl: string) {
+  private removeTrailingSlashURL(instanceUrl = '') {
```

Comment:
I'm not sure why we need this change since the `return instanceUrl ? ...` should check for null | undefined so we don't need a default parameter?

Reply:
This is one of those cases where i couldn't use an empty string as a default, because there is an undefined check on this property.
```
if (instanceUrl) {
      envVars[ENV_SFDX_INSTANCE_URL] = instanceUrl;
    }
```

I'll update the method to take ```string | undefined``` and let it's internals deal with the undefined.
```
private removeTrailingSlashURL(instanceUrl: string | undefined) {
    return instanceUrl ? instanceUrl.replace(/\/+$/, '') : '';
  }
```

Reply:
done